perses/rjmc/geometry.py

```diff
@@ -455,11 +467,10 @@ def _bond_logp(self, r, bond, beta):
         """
         k_eq = bond.type.k*units.kilocalories_per_mole/(units.angstrom**2)
         r0 = bond.type.req*units.nanometers
-        sigma = beta*2.0/np.sqrt(2.0*k_eq/k_eq.unit)
-        logp = stats.distributions.norm.logpdf(r/r.unit, r0/r0.unit, sigma)
-        return logp
+        logq = -beta*0.5*k_eq*(r-r0)**2
+        return logq
```

Comment:
If this bond is constrained, we should return a `logq` of 0.  We might also add a check whether `np.abs((r-r0)/r0)` is within some specified relative tolerance in that case, or throw an exception.